dir.props

```diff
@@ -68,6 +68,8 @@
     <ObjDir Condition="'$(ObjDir)'==''">$(BinDir)obj/</ObjDir>
     <ProductBinDir Condition="'$(ProductBinDir)'==''">$(BinDir)Product/</ProductBinDir>
     <TestWorkingDir Condition="'$(TestWorkingDir)'==''">$(BinDir)tests/</TestWorkingDir>
+    <PackageOutputRoot Condition="'$(PackageOutputRoot)'=='' and '$(NonShippingPackage)' == 'true'">$(BinDir)packages_noship/</PackageOutputRoot>
+    <PackageOutputRoot Condition="'$(PackageOutputRoot)'==''">$(ProductBinDir)pkg/</PackageOutputRoot>
```

Comment:
This line will always overwrite the PackageOutputRoot set on the previous line. Please either remove the previous line or fix this one's condition to include '$(NonShippingPackage)' != 'true'